build/ferretdb/eval-dev.Dockerfile

```diff
@@ -133,7 +134,7 @@ ENV GORACE=halt_on_error=1,history_size=2
 # do not allow "trust" authentication for local connections while initializing
 # ENV POSTGRES_INITDB_ARGS="--auth=scram-sha-256"
 
-ENV FERRETDB_STATE_DIR=/tmp/state
+ENV FERRETDB_STATE_DIR=/state
```

Comment:
Let's make it more like other images, where, for example, we have `FERRETDB_STATE_DIR` under `FERRETDB_DEBUG_ADDR`, so "# don't forget to update documentation if you change defaults" comment applies to it, too